scylla/templates/scylla-statefulset.yaml

```diff
@@ -58,9 +60,13 @@ spec:
             mountPath: /opt/ready-probe.sh
             subPath: ready-probe.sh
       volumes: # TODO: data is discarded if pod restarts, change to something more persistant 
-        - name: scylla-data 
-          emptyDir:
+        - name: scylla-data
+          {{ if .Values.statefulset.volume }}hostPath:
```

Comment:
Please separate this in two lines. Use `{{- if` if you don't want template lines to be printed